index.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/**
+ * PHP starts here.
+ */
+
+//Import the Field Class.
+require("./Fields.php");
+
+$fieldArray = fieldServices();
+
+/**
+ * Function fieldService()
+ * Call the api.
+ * Check if the field title is null or not.
+ * If not null then.
+ * Store the all different data in different variables.
+ * @var fieldTitle string : Stores the service field title. 
+ * @var fieldService string : Stores the all services URLs of a particular field.
+ * @var fieldImage string: Stores the image URL of a particular field.
+ * @var alias string: Stores the URL for particular field for more detail about that service.
+ * @var iconArr array: Stores URLs of the icons for a particular field of service.
+ * 
+ * @return fieldArray array: Stores the all fetched data in the form of array.
+ * 
+ */
+function fieldServices()
+{
+  $fieldArray = [];
+
+  $arrBody = (new FetchApi('https://www.innoraft.com/jsonapi/node/services'))->apiCall();
+
+  foreach ($arrBody['data'] as $data) {
+    $baseUrl = "https://www.innoraft.com"; // Set the base url.
+
+    /**
+     * Check of the service title is null or not.
+     * 
+     * If not null then fetch the all required data.
+     * 1. Field title.
+     * 2. Field services links.
+     * 3. Field images.
+     * 4. Respective self url of particular service.
+     * 5. Respective service icons.
+     * 
+     */
+    if (($data['attributes']['field_secondary_title']) != NULL) {
+      $iconsArr = [];
+      $fieldTitle = $data['attributes']['field_secondary_title']['value'];
+      $fieldService =  $data['attributes']['field_services']['value'];
+      $fieldImage = $baseUrl . (new FetchApi($data['relationships']['field_image']['links']['related']['href']))->apiCall()['data']['attributes']['uri']['url'];
+      $alias = $baseUrl . $data['attributes']['path']['alias'];
+      $fieldIcons = (new FetchApi($data['relationships']['field_service_icon']['links']['related']['href']))->apiCall()['data'];
+
+      for ($i = 0; $i < count($fieldIcons); $i++) {
+        $iconArray = $baseUrl . (new FetchApi($fieldIcons[$i]['relationships']['field_media_image']['links']['related']['href']))->apiCall()['data']['attributes']['uri']['url'];
+        $iconsArr[] = $iconArray;
+      }
+      $obj = new Field($fieldImage, $fieldTitle, $alias, $fieldService, $iconsArr);
+      $fieldArray[] = $obj;
```

Comment:
Couldn't we do this in one line, instead of using `$iconArray` directly store it in `$iconArr[]`?